staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go

```diff
@@ -37,7 +38,7 @@ import (
 // APIHandlerManager defines the behaviour that an API handler should have.
 type APIHandlerManager interface {
 	AddAPIService(apiService *apiregistration.APIService) error
-	RemoveAPIService(apiServiceName string)
+	RemoveAPIService(tenantedApiServiceName string)
```

Comment:
why the renaming? is this needed?

Reply:
it is not necessary, but it helps code reading. 

apiServiceName is APIService.Name
tenantedApiServiceName is APIService.Tenant + "/" + APIService.Name.

variable name "apiServiceName" is used everywhere. Renaming helps identifying the difference.